tests/general/LineCountBearTest.py

```diff
@@ -25,3 +26,32 @@ def test_run(self):
             filename='default')
         self.check_validity(self.uut, ['1 line'])
         self.check_validity(self.uut, [])  # Empty file
+
+    def test_exclude_blank_lines(self):
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('max_lines_per_file', 2))
+        self.section.append(Setting('min_lines_per_file', 1))
+        self.check_results(
+            self.uut, ['line 1', ' ', 'line 2',
+                       'line 3', '\n', '\t', ' line 4',
+                       'line 5 ', ' line 6 ', '\t\tline 7'],
+            [Result.from_values('LineCountBear',
+                                'This file had 7 lines, which is 5 lines more '
+                                'than the maximum limit specified.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_min_lines_per_file(self):
```

Comment:
Please add a test with `exclude_blank_lines` set to `False` in which this test file passes.